src/object/object_representation.c

```diff
@@ -281,11 +281,19 @@ int or_Type_sizes[] = {
  *    need to copy the string out of the record for some architectures
  *    if there are weird alignment or character set problems.
  */
-char *
-or_class_name (RECDES * record)
+int
+or_class_name (RECDES * record, char **string, int *alloced_string)
 {
-  char *start, *name;
-  int offset, len;
+  char *start = NULL;
+  int offset = 0;
+  int len = 0;
+
+  OR_BUF buffer;
+  int compressed_length = 0;
+  int decompressed_length = 0;
+  int rc = NO_ERROR;
+
+  assert (*alloced_string == 0);
```

Comment:
NULL check assertion
```suggestion
  assert (string != NULL);
  assert (alloced_string != NULL);
```
for the `assert (*alloced_string == 0);`, There is no need to check that this variable is initialized to zero.

Reply:
I've been talking about the same topic with @ctshim.
In conclusion, I decided to initialize it to NULL in or_class_name.

Since it is a variable passed as an argument to a function,
it seems to be recommended to initialize NULL before calling the function.
However, this increases unnecessary code, so I will initialize it to NULL and use it internally.

Reply:
When ` or_class_name` is called as the following, it should be checked on debug mode.
```
or_class_name (record, NULL, NULL);
```

Reply:
Yes, I added the two check codes below.
```
assert (string != NULL);
assert (alloced_string != NULL);
```